test/testall.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Run all unit tests for cclib."""
+
+import unittest
+
+import test_data
+
+from test_bridge import *
+from test_io import *
+from test_method import *
+from test_parser import *
+
+
+if __name__ == "__main__":
+	print "Running unit tests for data..."
```

Comment:
Yeah, we don't run all the tests in Travis CI - we skip bridge, I think, IIRC because I couldn't get openbabel to install cleanly.